src/components/PayPal/types.ts

```diff
@@ -12,7 +12,7 @@ declare global {
  * The intent for the transaction. This determines whether the funds are captured immediately, or later.
  * @see {@link https://developer.paypal.com/docs/checkout/reference/customize-sdk/#intent}
  */
-type Intent = 'sale' | 'capture' | 'authorize' | 'order';
+type Intent = 'sale' | 'capture' | 'authorize' | 'order' | '';
```

Comment:
I'd undo this change in order to make the `Intent` type more precise.
I know we have `intent: ''` in a couple of places, but we could replace the empty string with `null`.

Reply:
done